source/api/extension/navplace/index.md

```diff
@@ -261,7 +260,7 @@ Here you can see an example of a IIIF Manifest with the `navPlace` property. It
             "properties":{
                "label":{
                   "en":[
-                     "Photograph from Göttingen, Germany"
+                     "navPlace Extension Specification Example Manifest"
```

Comment:
If we aren't dodging properties, then our full example should have one.  I am repeating the label property of the Manifest here now (as opposed to the Canvas, b/c navPlace is on the Manifest not the Canvas).  This is b/c of how we worded 2.2 and 3.3 with "MAY be repeated".